### ISSUE TITLE
Overfitting and Finetuning

### ISSUE THREAD

**Issue by aliosia** [minimaxir/aitextgen#136]
Hi. During the finetuning process, the model may forget some knowledge and just focus on the target style. Is there any option to set validation data and track its loss? if not, how should I interpret training loss and what is your recommendation about the size of training data, number of steps (default: 5000) and learning rate (default: 0.001)? Thanks

**Comment by minimaxir** [comment]
Overfitting is typically a function of not having enough training data. Typically you want atleast 1MB of text data.

If you have less, you'll need to use fewer steps and a lower learning rate.

Unfortunately, it's not an exact science, even if validation support were added.
